tools/StaticAnalysis/Exceptions/ValidateHelpIssues.csv

```diff
@@ -0,0 +1,142 @@
+Target,Description
+Get-AzApiManagementProperty.md,No synopsis found
+Get-AzApiManagementProperty.md,No description found
+Remove-AzApiManagementAuthorizationServer.md,No examples found
+Update-AzApiManagementRegion.md,No examples found
+Remove-AzAutomationDscConfiguration.md,No examples found
+Remove-AzAutomationDscNodeConfiguration.md,No examples found
```

Comment:
Any reason to add this file? Please remove it.

Reply:
removed